components/scripts/gradle/03-validate-local-build-caching-different-locations.sh

```diff
@@ -19,6 +19,7 @@ readonly SCRIPT_NAME
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")"; cd -P "$(dirname "$(readlink "${BASH_SOURCE[0]}" || echo .)")"; pwd)"
 readonly SCRIPT_DIR
 readonly LIB_DIR="${SCRIPT_DIR}/lib"
+readonly SCANS_SUPPORT_TOOLS_JAR="${SCRIPT_DIR}/scans-support-tools.jar"
```

Comment:
Please rename to `build-scan-support-tool.jar`. And also adjust the name in the error message when the jar is not there.

Reply:
Changed in 959de12707c38c55194c27955284d8a8bc5830ca